dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -10,16 +10,16 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
 
 MENU_001 MENUEX
 BEGIN
-    MENUITEM "Picto&grame mari", FCIDM_SHVIEW_BIGICON
-    MENUITEM "&Pictograme mici", FCIDM_SHVIEW_SMALLICON
-    MENUITEM "&Listă", FCIDM_SHVIEW_LISTVIEW
-    MENUITEM "&Detalii", FCIDM_SHVIEW_REPORTVIEW
+    MENUITEM "Dal&e", FCIDM_SHVIEW_BIGICON
+    MENUITEM "&Miniaturi", FCIDM_SHVIEW_SMALLICON
+    MENUITEM "&Listare", FCIDM_SHVIEW_LISTVIEW
+    MENUITEM "D&etalii", FCIDM_SHVIEW_REPORTVIEW
```

Comment:
It must be &Detalii according to your MS pic. That is important to prevent it colliding with Dal&e! Please change!
Keep in mind that there is also IDS_VIEW_DETAILS "&Detalii" on the bottom of the file, where it is already correct.

Reply:
Kind of resolved. Collision is resolved now. I didn't request to change IDS_VIEW_DETAILS "&Detalii" to IDS_VIEW_DETAILS "Detalii" though. But I don't care. At least there is no collision anymore.